src/Sylius/Bundle/WebBundle/Resources/config/routing/backend/shipping_method.yml

```diff
@@ -54,3 +54,11 @@ sylius_backend_shipping_method_show:
         _controller: sylius.controller.shipping_method:showAction
         _sylius:
             template: SyliusWebBundle:Backend/ShippingMethod:show.html.twig
+
+sylius_backend_shipping_method_delete:
+    path: /{id}
+    methods: [DELETE]
+    defaults:
+        _controller: sylius.controller.shipping_method:deleteAction
+        _sylius:
+            redirect: sylius_backend_shipping_method_index
```

Comment:
Every other delete routing uses explicit redirect to index in their configs. There is a default `redirectToIndex` method in ResourceController, but it doesn't seem to work without configuration.